pkg/daemon/ceph/client/mgr.go

```diff
@@ -35,8 +35,8 @@ func MgrDisableModule(context *clusterd.Context, clusterName, name string) error
 }
 
 // MgrSetAllConfig applies a setting for all mgr daemons
-func MgrSetAllConfig(context *clusterd.Context, clusterName string, cephVersion cephver.CephVersion, key, val string) (bool, error) {
-	return MgrSetConfig(context, clusterName, "", cephVersion, key, val, false)
+func MgrSetAllConfig(context *clusterd.Context, clusterName string, mgrName string, cephVersion cephver.CephVersion, key, val string) (bool, error) {
```

Comment:
Since this changed for a single mgr, could you rename this to `MgrSetConfig` and update the comment?

Reply:
done